test/scripts/test_clicReconstruction.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
```

Comment:
Is it possible to add to fail if any command fails, or if there is an undefined variable?
```suggestion
set -eu
```